ahpi.py

```diff
@@ -91,12 +92,18 @@ def __build_abholungen(gemeinde, street, hausnr, loading_place):
 
     return return_object
 
-def get_abholungen(gemeinde, input_street, hausnr, loading_place):
+def get_abholungen(input_gemeinde, input_street, hausnr, loading_place):
+    gemeinde = input_gemeinde.lower().capitalize()
     raw_streets = get_streets(gemeinde, input_street[0])
-    street = [ street for street in raw_streets if input_street in street ][0]
+    matching_streets = [ street for street in raw_streets if re.search(input_street, street, re.IGNORECASE) ]
+    matching_streets_count = len(matching_streets)
+    street = matching_streets[0] if matching_streets_count == 1 else usage_error("No matching street found") if matching_streets_count == 0 else usage_error("Street is ambigous: " + ", ".join(matching_streets))
```

Comment:
I know that my code isn't that well formatted, but would it be possible to split this up into multiple lines instead of one big line? :)

Reply:
You are right. This doesn't read well.

I've ran the code through [Black](https://github.com/psf/black) which makes it look pretty nice in my opinion.

Reply:
Thanks, looks good to me. Black seems like a good idea, i will format the code in one of the following commits :)